mods/cnc/rules/aircraft.yaml

```diff
@@ -47,7 +47,7 @@ HELI:
 		Cost: 1200
 	Tooltip:
 		Name: Apache Longbow
-		Description: Helicopter Gunship with Chainguns.\n  Strong vs Infantry, Light Vehicles\n  Weak vs Tanks
+		Description: Helicopter gunship with chainguns.\n  Strong vs infantry, light vehicles &\n  aircraft.\n  Weak vs tanks.
```

Comment:
`Strong vs Infantry, Light vehicles, Aircraft\n  Weak vs Tanks`, to be consistent with #8277 and #7967, please.